schedules/backfill_schedule.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+from datetime import datetime, timedelta
+
+from temporalio.client import Client, ScheduleBackfill, ScheduleOverlapPolicy
+
+
+async def main():
+    client = await Client.connect("localhost:7233")
+    handle = client.get_schedule_handle(
+        "workflow-schedule-id",
+    )
+    now = datetime.utcnow()
+    if now.second == 0:
+        now += timedelta(seconds=1)
+    await handle.backfill(
```

Comment:
Backfilling a schedule for just "now" makes little sense. You want to backfill in the past. And usually over a range.